app/javascript/listings/__tests__/ListingDashboard.test.jsx

```diff
@@ -99,258 +102,286 @@ global.document.body.innerHTML = `<div id="listings-dashboard" data-listings=${J
+
+global.document.body.innerHTML = `<div id="listings-dashboard" data-listings=${JSON.stringify(
+  listingsForDataAttribute,
+)} data-usercredits="3" data-orglistings=${JSON.stringify(
+  orgListing,
+)} data-orgs=${JSON.stringify(orgs)} ></div>`;
 global.window = doc.defaultView;
 
 const listings = {
-  listings: [
-    {
-      id: 23,
-      bumped_at: '2019-06-11T16:45:37.229Z',
-      category: 'cfp',
-      location: 'New York City',
-      organization_id: null,
-      slug: 'asdfasdf-2ea8',
-      title: 'asdfasdf',
-      updated_at: '2019-06-11T16:45:37.237Z',
-      user_id: 11,
-      tag_list: [Array],
-      author: [Object],
-    },
-    {
-      id: 24,
-      bumped_at: '2019-06-11T16:59:16.312Z',
-      category: 'events',
-      location: 'Denver',
-      organization_id: 2,
-      slug: 'yoyoyoyoyoooooooo-4jcb',
-      title: 'YOYOYOYOYOOOOOOOO',
-      updated_at: '2019-06-11T16:59:16.316Z',
-      user_id: 11,
-      tag_list: [Array],
-      author: [Object],
-    },
-    {
-      id: 25,
-      bumped_at: '2019-06-11T17:01:25.143Z',
-      category: 'cfp',
-      location: 'Seattle',
-      organization_id: 3,
-      slug: 'hehhehe-5hld',
-      title: 'hehhehe',
-      updated_at: '2019-06-11T17:01:25.169Z',
-      user_id: 11,
-      tag_list: [],
-      author: [Object],
-    },
-  ],
-  orgListings: [
-    {
-      id: 24,
-      bumped_at: '2019-06-11T16:59:16.312Z',
-      category: 'events',
-      location: 'Denver',
-      organization_id: 2,
-      slug: 'yoyoyoyoyoooooooo-4jcb',
-      title: 'YOYOYOYOYOOOOOOOO',
-      updated_at: '2019-06-11T16:59:16.316Z',
-      user_id: 11,
-      tag_list: [Array],
-      author: [Object],
-    },
-    {
-      id: 25,
-      bumped_at: '2019-06-11T17:01:25.143Z',
-      category: 'cfp',
-      location: 'Seattle',
-      organization_id: 3,
-      slug: 'hehhehe-5hld',
-      title: 'hehhehe',
-      updated_at: '2019-06-11T17:01:25.169Z',
-      user_id: 11,
-      tag_list: [],
-      author: [Object],
-    },
-  ],
-  orgs: [
-    { id: 2, name: 'Yoyodyne', slug: 'org3737', unspent_credits_count: 1 },
-    { id: 3, name: 'Infotrode', slug: 'org5254', unspent_credits_count: 1 },
-  ],
+  listings: listingsForDataAttribute,
+  orgListings: orgListing,
+  orgs,
   userCredits: '3',
   selectedListings: 'user',
 };
 
 /* eslint-disable no-unused-vars */
 /* global globalThis timestampToLocalDateTimeLong timestampToLocalDateTimeShort */
 
+const setup = () => {
+  return render(<ListingDashboard />);
+};
+
 describe('<ListingDashboard />', () => {
   afterAll(() => {
     delete globalThis.timestampToLocalDateTimeLong;
     delete globalThis.timestampToLocalDateTimeShort;
   });
 
-  it('should have no a11y violations', async () => {
-    const { container } = render(<ListingDashboard />);
-    const results = await axe(container);
+  beforeEach(setup);
 
-    expect(results).toHaveNoViolations();
-  });
+  describe('Acessbility check', () => {
+    beforeAll(cleanup);
```

Comment:
Do we need this? I thought testing library automatically ran cleanup after each test 🤔